utils/misc/misc.go

```diff
@@ -274,30 +274,42 @@ func AddFileToZip(zipWriter *zip.Writer, filename string) error {
 
 	writer, err := zipWriter.CreateHeader(header)
 	if err != nil {
-		panic(err)
+		return err
 	}
 	_, err = io.Copy(writer, fileToZip)
 	return err
 }
 
 // UnZipSingleFile unzips zip containing single file into ouputfile path passed
-func UnZipSingleFile(outputfile string, filename string) {
+func UnZipSingleFile(outputFile string, filename string) error {
```

Comment:
If this function is not used I would propose to delete it. It doesn't have any tests and they are things to be fixed anyway.

Reply:
removed